ui/public/distro-res/strings.json.example

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I think you can put this example in https://github.com/PingCAP-CBG/tidb-dashboard-distro